sdk/appconfiguration/app-configuration/src/internal/cryptoHelpers.browser.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @internal
+ * @ignore
+ */
+export async function sha256Digest(body: any): Promise<string> {
```

Comment:
How are we ensuring test coverage? I would imagine live tests running in browser would suffice right? Do we have those?

Reply:
Yes, I definitely want to add them - I wasn't sure how difficult it was so I didn't address it in this particular PR but you're 100% right.

Reply:
Try setting `TestBrowser` to true at https://github.com/Azure/azure-sdk-for-js/blob/184753c43d79e443331d1b1a19e300ecd0aee5f0/sdk/appconfiguration/app-configuration/tests.yml#L8

@KarishmaGhiya Should that be enough?

Reply:
Just added it, now we'll see those browser tests passing! (hopefully)

Reply:
And they passed!